app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ProfilesController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def show_me
+    @user = User.find(current_user.id)
```

Comment:
Redundant: `current_user` is equal to `User.find(current_user.id)`.

Reply:
resolved in https://github.com/hpi-swt2/bookkeeper-portal-red/pull/44/commits/30f3e8fb4cc7dea36d12bb49d696821e5efb476b